routing/missioncontrol.go

```diff
@@ -93,6 +93,10 @@ type NodeResults map[route.Vertex]TimedPairResult
 // since the last failure is used to estimate a success probability that is fed
 // into the path finding process for subsequent payment attempts.
 type MissionControl struct {
+	// UpdateEstimator is a function that is called whenever the
+	// mission control state is updated.
+	UpdateEstimator func(estimator Estimator)
```

Comment:
nit: since this field only represents the internal state of the `MissionControl` struct and doesn't need to be set or read outside of this file, it doesn't need to be exported (lowercase vs. uppercase in Golang). So can just be `updateEstimator`.